MAINTENANCE.md

```diff
@@ -0,0 +1,80 @@
+# Roles, People, and Processes for Maintaining the Tagging Schema
+
+This document outlines how this project is maintained.
+
+## Roles & People
+
+### Maintainer Role
+
+[Martin](https://github.com/tyrasd) maintains this project as part of his work on the iD Editor project.
+
+The maintainer role includes:
+- Having the final say in decisions for the tagging schema.
+- Creating releases.
+- Updating dependencies.
+
+and all the following roles.
+
+### Contributor Role
+
+- [Kyle](https://github.com/k-yle) contributes to this project as a volunteer.
+- [Tobias](https://github.com/tordans) contributes to this project as a volunteer.
+
+The contributor role includes:
+- Reviewing PRs.
+- Merging "clear-cut" PRs.
+
+and all the following roles.
+
+### Triage Role
+
+(No one has the triage role at the moment. Please reach out if you want to help.)
+
+The triage role includes:
+- Helping to clarify issues and PRs.
+- Closing issues as duplicates or not planned.
+
+### Contributors
+
+To all contributors, thank you so much for your support! ❤ Especially for:
+- Suggesting new presets and fields or updates to the repository.
+- Researching and helping with issues and PRs.
+- Translating the tagging schema.
+
+Code contributions: [Check this complete list of contributors on GitHub](https://github.com/openstreetmap/id-tagging-schema/graphs/contributors).
+
+## Processes
+
+### PR Reviews and Merges
+
+- PRs need approval from two people: the author and one or more contributors before being merged.
+- Non-"clear-cut" changes need to be merged by the maintainer.
+- We might revert merges later if necessary.
+
+**What is a clear-cut change?**
+
+- No or minimal controversial discussion on the change.
+- Coding and contribution [guidelines](./GUIDELINES.md) are met.
+
+**How to merge…**
+
+- Usually squash merge PRs to make the history simpler
+- Give the merge a meaningful description of the change
+- Add labels to the PR to simplify creating the changelog: `new-*`, `add-*`, `bug`
+
+### Releases
+
+There is no set release schedule at the moment, but releases usually occur every other month. After this project is released, the projects that rely on the data need to update and release as well.
+
+### Assigning roles
+
+- The maintenance role is assigned by the OSMF. Historically the maintainer of the iD Editor also maintained this project.
```

Comment:
```suggestion
- The maintenance role is assigned by the OSMF. So far the maintainer of the iD Editor also maintained this project.
```

"historically" is typically used in cases where it was applying in past but is not any more, and implies such situation. At least as far as I know, I am not a native speaker.

(BTW, is it it "iD editor" or "iD Editor"?)

Reply:
I see your point. I asked ChatGPT for some wording and used parts of that to change the sentence. I also removed the mention of OSMF which is something that the iD Repo can explain better.

Reply:
Will resolve this so we can create a new comment on the new line.